.github/workflows/release.yml

```diff
@@ -1,37 +1,112 @@
-name: Publish package to the ForgeRock artifactory
+name: create_release_workflow
+run-name: Create release final::${{ inputs.is_final_release }} candidate::${{ inputs.candidate_number }}
+# **What it does**:
+# Reuse workflow 'prepare-release' from parent: update versions, create tag and release branch with release versions
+# Build the maven project as release version and publish the artifact in forgerock maven repo
+# **Artifacts**: java libraries and bill of materials
+# **Artifact repository**: https://maven.forgerock.org/artifactory/community/com/forgerock/sapi/gateway/
 on:
-  release:
-    types: [published]
+  workflow_dispatch:
+    inputs:
+      notes:
+        description: "Release notes"
+        required: false
+        type: string
+        default: ''
+      is_final_release:
+        description: "Is a final release, otherwise will be a 'rc' (release candidate)"
+        required: false
+        type: boolean
+        default: false
+      candidate_number:
+        description: "Provide the candidate sequence number (default: 1)"
+        required: false
+        type: string
+        default: 1
+env:
+  GH_TOKEN: ${{ github.token }}
+  GIT_AUTHOR_NAME: ${{ secrets.GIT_COMMIT_USERNAME_BOT }}
+  GIT_AUTHOR_EMAIL: ${{ secrets.GIT_COMMIT_AUTHOR_EMAIL_BOT }}
+  GIT_COMMITTER_NAME: ${{ secrets.GIT_COMMIT_USERNAME_BOT }}
+  GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMIT_AUTHOR_EMAIL_BOT }}
+
 jobs:
-  publish:
+
+  release_prepare: # prepare for a release in scm, creates the tag and release branch with the proper release versions
+    name: Call release prepare
+    # uses: ./.github/workflows/release-prepare.yml # same repository
+    uses: SecureApiGateway/secure-api-gateway-parent/.github/workflows/release-prepare.yml@issue/812-action-release-prepare-workflow
+    with:
+      is_final_release: ${{ inputs.is_final_release }}
+      candidate_number: ${{ inputs.candidate_number }}
+    secrets: # inherit only when is the same repository
+      GPG_PRIVATE_KEY_BOT: ${{ secrets.GPG_PRIVATE_KEY_BOT }}
+      GPG_KEY_PASSPHRASE_BOT: ${{ secrets.GPG_KEY_PASSPHRASE_BOT }}
+      GIT_COMMIT_USERNAME_BOT: ${{ secrets.GIT_COMMIT_USERNAME_BOT }}
+      GIT_COMMIT_AUTHOR_EMAIL_BOT: ${{ secrets.GIT_COMMIT_AUTHOR_EMAIL_BOT }}
```

Comment:
The `release_prepare` job is pointed currently to the `release-prepare.yml` branch `issue/812-action-release-prepare-workflow` just in case changes are needed in release prepare workflow that lives in the parent repository.